react/features/polls/reducer.ts

```diff
@@ -83,21 +83,37 @@ ReducerRegistry.register<IPollsState>('features/polls', (state = INITIAL_STATE,
         // if the poll exists, we update it with the incoming answer
         const newAnswers = state.polls[pollId].answers
             .map(_answer => {
+                const voters = _answer.voters as string[];
+
                 return {
                     name: _answer.name,
-                    voters: new Map(_answer.voters)
+                    voters: typeof voters?.length === 'number' ? [ ...voters ] : { ...voters }
                 };
             });
 
-        for (let i = 0; i < newAnswers.length; i++) {
-            // if the answer was chosen, we add the sender to the set of voters of this answer
-            const voters = newAnswers[i].voters;
-
-            if (answer.answers[i]) {
-                voters.set(answer.voterId, answer.voterName);
 
+        for (let i = 0; i < newAnswers.length; i++) {
+            // if the answer was chosen, we add the senderId to the array of voters of this answer
+            const voters = newAnswers[i].voters as any;
+
+            if (typeof voters?.length === 'number') {
```

Comment:
Yeah all this looks odd. Let's convert the old format to the new one and then only process that please.